cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -148,4 +184,5 @@ def describeType(DescribableHelper.ParameterType type, int headerLevel) throws E
             assert false: type;
         }
     }.call()
-}
\ No newline at end of file
+}
+st.adjunct(includes: 'org.jenkinsci.plugins.workflow.cps.Snippetizer.js.workflow')
```

Comment:
Oh I see, you have _another_ adjunct at line 15. You only need one to include all associated resources. :bug: